indico/modules/events/abstracts/blueprint.py

```diff
@@ -49,6 +50,7 @@
 _bp.add_url_rule('/manage/abstracts/open', 'open_abstracts_call', RHOpenCFA, methods=('POST',))
 _bp.add_url_rule('/manage/abstracts/close', 'close_abstracts_call', RHCloseCFA, methods=('POST',))
 _bp.add_url_rule('/manage/abstracts/boa', 'manage_boa', RHManageBOA, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/abstracts/boa.pdf', 'export_boa', RHExportBOA, methods=('GET', 'POST'))
```

Comment:
We just need `GET` for this endpoint since we are not sending any form data. You can omit `methods` altogether since `GET` is the default.